### DISCUSSION TITLE
Query regarding workflow execution

### DISCUSSION THREAD

**Discussion by nehapapat** [Netflix/conductor#2767] [Q&A]
Hi All,

I have a query regarding workflow execution. I have a condition where I want to get the api result from sub-workflow and utilize it in parent workflow to handle decisionCases in parent workflow.

**EXAMPLE:**

**Parent:**
[
  {
    "name": "callAPI",
    "taskReferenceName": "call api",
    "type": "SUB_WORKFLOW",
    "inputParameters": {
      "companyId": "${workflow.input.inputId}"
    },
    "subWorkflowParam": {
      "name": "GET_TOKEN",
      "version": 1
    }
  },
  {
    "name": "decide_task",
    "taskReferenceName": "validate_cancel_sub",
    "inputParameters": {
      "case_value_param": "${assert_sub_cancel_success.output.result}"
    },
    "type": "DECISION",
    "caseValueParam": "case_value_param"
  }
]
**Sub-workflow:**
[
  {
    "name": "GET_TOKEN",
    "description": "fetch token",
    "version": 1,
    "schemaVersion": 2,
    "tasks": [
      {
        "name": "get_token",
        "taskReferenceName": "get_token",
        "inputParameters": {
          "http_request": {
            "uri": "",
            "method": "POST",
            "accept": "application/json"
          }
        },
        "type": "HTTP",
        "startDelay": 0,
        "optional": false,
        "outputParameters": {
                  "output_find_active_subscriptions": "${get_token.output.response.body.access_token}"
         },
      }
    ]
  }
]

Here, I want to understand how can I retrieve result/output from sub-workflow back to parent workflow ?

Any assistance you can provide would be greatly appreciated.

**Comment by jxu-nflx** [comment]
Hello @nehapapat, the output of the sub workflow will be the task's output of the parent workflow (the "call api" task in this case). you could use the parent task's output (i.e. the "call api" task) the same way as any other tasks in the following `decide_task` task in parent workflow
